### ISSUE TITLE
How can this Divvy research assist our project?

### ISSUE THREAD

**Issue by Oboi** [zorostang/chicago_bike_equity#10]
"Where in Chicago Would Benefit From A Divvy Station?" by DePaul University student Ryan Adriatico assesses public transit deserts in Chicago and identifies which neighborhoods would benefit from the exansion of the Divvy bike share program. 

His map can be found here: http://via.library.depaul.edu/mom/20/

There is a gallery of other student maps here: http://via.library.depaul.edu/mom/ and http://via.library.depaul.edu/cgis/

For more information about the Map of the Month and Geographic Information Systems (GIS) at DePaul, please contact geography@depaul.edu

Shared by... 
Prof. Euan Hague
Chair, Department of Geography
EHAGUE@depaul.edu

**Comment by meemking** [comment]
This is really nice. Thanks for passing along. 

Whats the most up to date source of data in Chicago on volume of bikers by region/street/whatever in the city? Divvy must cite something when it decides where to put bike racks, right?

**Comment by stevevance** [comment]
@meemking The Chicago Department of Transportation posts bike count data (in its extremely limited, minimal, hard to use form) [on Scribd](https://www.scribd.com/collections/4289227/CDOT-Bike-Counts).

Each one is a PDF. To get a breakdown of counts by time you'll have to submit a FOIA request to CDOT.

**Comment by Oboi** [comment]
FYI @meemking & @stevevance ... 
CDOT asked Slow Roll Chicago to make outreach into the Southside and Westside to get folks to visit suggest.divvybikes.com in order to suggest new stations for the areas where they are expanding. The understanding here is CDOT will take this into consideration when deciding on the new locations. We are running a social media campaign for this through Feb 28. Check it out in our last email newsletter here: 
http://eepurl.com/bbN90P 
Also, I am mtg with CDOT on Tue. Let me know in case I can ask for any specific data which you all will find helpful.
Thanks, peace and be well...
Oboi

**Comment by meemking** [comment]
I think it would be critical to get CDOT's answer on the following, re
data:
1- What data do they use to decide where new stations go? Is it based
solely on requests from the public (ie the feedback on this social media
campaign)? Do they also take into account the volume of bikers already in
the area?
2- If so, what is the data source they use for counting the number of
bikers in the city, and how granular is this data? Ie, do they have an idea
of the number of bikers every hour on every intersection? Or just the
number of bikers every day in each neighborhood?
3- What is the coverage of the data above? Do they have a real time feed
from some sensors on the roads that continually feed information on volume
of bikers? Is the coverage equal throughout the city (are they able to
count bikers on the south side as well as on the north side?)

On Sun, Feb 8, 2015 at 11:15 AM, Olatunji Oboi Reed <
notifications@github.com> wrote:

> FYI @meemking https://github.com/meemking & @stevevance
> https://github.com/stevevance ...
> CDOT asked Slow Roll Chicago to make outreach into the Southside and
> Westside to get folks to visit suggest.divvybikes.com in order to suggest
> new stations for the areas where they are expanding. The understanding here
> is CDOT will take this into consideration when deciding on the new
> locations. We are running a social media campaign for this through Feb 28.
> Check it out in our last email newsletter here:
> http://eepurl.com/bbN90P
> Also, I am mtg with CDOT on Tue. Let me know in case I can ask for any
> specific data which you all will find helpful.
> Thanks, peace and be well...
> Oboi
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/zorostang/chicago_bike_equity/issues/10#issuecomment-73420352
> .

## 

Maggie King
email: Maggiek@uchicago.edu
phone: (574) 286-9389

**Comment by Jmamdor** [comment]
@meemking If I remember correctly, Divvy uses a proprietary model to site their stations. The suggestions and input from the city only impact that inasmuch as they need to nail down exact spot. The model uses employment, residential, and recreational attraction density. It also contains a financial impact portion to determine if sites are economical.

Steven would know more about the other two. I think the general consensus it that the city seldom counts cyclists at all, and when they do, they stink it up.

**Comment by meemking** [comment]
Thanks @jmamdor :)  Maybe the right question then is how financial impact
is modeled. Basic assumption is that it would require some input
assumptions about the number of bikers using that station. Without a good
estimate of volume of bikers, how are they arriving at that impact?

On Mon, Feb 9, 2015 at 3:01 PM, Jmamdor notifications@github.com wrote:

> @meemking https://github.com/meemking If I remember correctly, Divvy
> uses a proprietary model to site their stations. The suggestions and input
> from the city only impact that inasmuch as they need to nail down exact
> spot. The model uses employment, residential, and recreational attraction
> density. It also contains a financial impact portion to determine if sites
> are economical.
> 
> Steven would know more about the other two. I think the general consensus
> it that the city seldom counts cyclists at all, and when they do, they
> stink it up.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/zorostang/chicago_bike_equity/issues/10#issuecomment-73589262
> .

## 

Maggie King
email: Maggiek@uchicago.edu
phone: (574) 286-9389

**Comment by bonfirefan** [comment]
Summary: Ryan Adriatico looked at a neighborhood level and highlighted those with 25%+  of the population below the poverty line. He then mapped rail stations and Divvy stations (not bus) with 0.5 mi buffers, with the remaining neighborhood areas highlighted as transit deficient. 

Gino and I discussed this after reading through the description - this research project in particular could serve as a possible starting point or model for starting the Divvy discussion but doesn't contain any hard data for our purposes. Additionally, we want to avoid the correlation with population 
density. Looking at census tract level data and doing the same analysis (perhaps with bus stops included) might reflect transit "holes" better. We also put forward the idea of looking at methodologies applied in food desert research. 

In short: we can make a similar map that will tell us more. Knowing more about the selection process will definitely help.

**Comment by zorostang** [comment]
Good summary @bonfirefan !

**Comment by stevevance** [comment]
@meemking No. 2: This is the question I, as a transportation planner in Chicago, get asked the most. And the answer is pretty insufficient, and doesn't help the person asking. And I can't say that the answer gets any better. There are few counts conducted in few places and for a few hours. Essentially the city counts in 6 locations 12 times a month and 6 hours per month. That's 432 hours per year, or 4.9% of the year. But at least they have consistency going for them!

**Comment by Oboi** [comment]
Peace All - Had a great mtg yesterday with CDOT. Anything y'all need from Divvy, let me know. Happy to ask CDOT. Our campaign on Divvy + Equity Talk goes through the end of Feb. I am confident CDOT will take this into consideration for new stations on the South & West sides. The more we get people to chime in the, better our position to speak to CDOT/Divvy from a place of power. 

Here's the text from the campaign...

Divvy + Equity Talk... 

Join the chorus, elevate your voice, and help make equity a priority at the Chicago Department of Transportation's Divvy program (https://www.divvybikes.com/about). Divvy is expanding in Chicago, adding more stations on the Southside and Westside. Now is the time to contribute to the decision making process for where the incoming stations will be located. 

Please take a few moments to suggest new locations for additional Divvy stations, by going to this WEBSITE (http://suggest.divvybikes.com/page/about). Click on the blue "SUGGEST A LOCATION" button at the bottom of the map. Notice the red & white tear drop with a + symbol in the middle appears approximately in the center of the map. Now, you can zoom in/out and move the map to situate the red & white tear drop where you want the new Divvy station to be located. Once you have the map oriented, notice the white dialogue box immediately to the right of the map. Please take a moment to completely fill out the information being requested. We are especially asking Southside and Westside slow rollers to complete this information (name, email, description, and zip code). With this information submitted, we will request CDOT publicly provide a report and analysis of this data and publicly share their approach to selecting new locations for Divvy stations based on this data.

After you complete the form, hit the blue "SUBMIT" button at the bottom. Repeat these steps for any additional locations you will like to suggest for Divvy stations. NOTE: Please suggest stations even where previously suggested locations already exist. This will allow us and CDOT to more accurately measure our impact. 

May our collective voices ring out, advocating on behalf of a diverse, equitable bicycle culture in Chicago and making equity a priority at CDOT's Divvy program...
SUGGEST.DIVVYBIKES.COM
